### PR TITLE
Remove hardcoded fields for parameter substitution configuration in Cluster Loader

### PR DESCRIPTION
**Pull request by sjug**
Fixes https://github.com/openshift/origin/issues/17068

This should enable us to remove `fatih/structs` as a vendor dep as well.

### CODE REVIEW

**Comment by deads2k** [comment]
Targeted for 3.8?

**Comment by sjug** [comment]
Whenever the soonest we can get it in for ideally @deads2k

**Comment by sjug** [comment]
@deads2k Can we please merge this is for 3.8, there are follow-up PRs that have this as a dependency.

**Comment by deads2k** [comment]
> @deads2k Can we please merge this is soon? There are follow-up PRs that have this as a dependency.

Well, the queue reopened, but the rebase is in progress and this hits godeps.json.

/approve
/hold

@soltysh I've held for you.  Release when you think it is safe.

**Comment by deads2k** [comment]
discussed on irc.

/hold cancel

**Comment by soltysh** [comment]
@sjug now that we have glide (see #17391) I think you need to update `glide.lock` as well here. /cc @deads2k

**Comment by sjug** [comment]
@soltysh I updated the `glide.lock` and squashed it into the vendor drop commit

**Comment by sjug** [comment]
@soltysh Is this alright?

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1557,11 +1557,6 @@
 			"ImportPath": "github.com/fatih/camelcase",
 			"Rev": "f6a740d52f961c60348ebb109adde9f4635d7540"
 		},
-		{
```
Comment (deads2k) [inline Godeps/Godeps.json]: don't touch this file

**Comment by deads2k** [comment]
glide looks as expected, but we're planning on branching 3.8 early tomorrow.

/hold

**Comment by sjug** [comment]
Can I please get a LGTM?

**Comment by soltysh** [comment]
@sjug this is ok, but now that we're close to landing a reabse any vendor-related changes are on-hold. Ping me after the rebase PR lands (#17576) and I'll push this forward.

**Comment by sjug** [comment]
@soltysh PR landed, please push

**Comment by soltysh** [comment]
@deads2k's https://github.com/openshift/origin/pull/17072#discussion_r154201247 still applies. Drop the `Godeps.json` change and you're good to go.

**Comment by sjug** [comment]
@soltysh Dropped the `Godeps.json` change.

**Comment by sjug** [comment]
[FATAL] Required `go` binary was not found in $PATH.

Damn I forgot to install `go`...

**Pull request merged**